cluster/images/etcd-empty-dir-cleanup/Dockerfile

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gliderlabs/alpine
+FROM busybox
+
+COPY etcdctl etcd-empty-dir-cleanup.sh /
+RUN chmod a+rx /etcdctl /etcd-empty-dir-cleanup.sh
 
-RUN apk-install bash
-ADD etcd-empty-dir-cleanup.sh etcd-empty-dir-cleanup.sh
-ADD etcdctl etcdctl
 ENV ETCDCTL /etcdctl
 ENV SLEEP_SECOND 3600
-RUN chmod +x etcd-empty-dir-cleanup.sh
-CMD bash /etcd-empty-dir-cleanup.sh
+
+# nobody:nobody
+USER 65534:65534
```

Comment:
I think this conditional might be wrong:
```bash
if [[ $(${ETCDCTL} ls $1) ]]; then
```

it happens to work in bash, but I'm not sure what it's actually trying to test.